utils/gyb_syntax_support/AttributeNodes.py

```diff
@@ -197,5 +199,122 @@
          ]),
 
     # objc-selector -> objc-selector-piece objc-selector?
-    Node('ObjCSelector', kind='SyntaxCollection', element='ObjCSelectorPiece')
+    Node('ObjCSelector', kind='SyntaxCollection', element='ObjCSelectorPiece'),
+
+    # The argument of '@differentiable(...)'.
+    # differentiable-attr-arguments ->
+    #     differentiation-params-clause? ','?
+    #     differentiable-attr-func-specifier? # jvp
+    #     differentiable-attr-func-specifier? # vjp
+    #     generic-where-clause?
+    Node('DifferentiableAttributeArguments', kind='Syntax',
+         description='''
+         The arguments for the `@differentiable` attribute: an optional \
```

Comment:
This description string should be fixed. Currently, it looks like this in [`Sources/SwiftSyntax/gyb_generated/syntax_nodes/SyntaxNodes.swift`](https://github.com/apple/swift-syntax/pull/178/files#diff-be6fd49b9da56aa4d28b732c3f377cd8R6904):

```swift
/// 
/// The arguments for the `@differentiable` attribute: an optional          differentiation parameter list and associated functions.
/// 
public struct DifferentiableAttributeArgumentsSyntax: SyntaxProtocol, SyntaxHashable {
  ...
```
A corresponding change should be made to [apple/swift-syntax](https://github.com/apple/swift-syntax) to update gyb-generated sources, e.g. https://github.com/apple/swift-syntax/pull/178.